changelog-2022.07.27-3.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/qt/res/app.organicmaps.desktop.metainfo.xml b/qt/res/app.organicmaps.desktop.metainfo.xml
+index 9174f31020..eece37e24d 100644
+--- a/qt/res/app.organicmaps.desktop.metainfo.xml
++++ b/qt/res/app.organicmaps.desktop.metainfo.xml
+@@ -107,6 +107,22 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.07.27-3" date="2022-07-27">
++     <description>
++       <ul>
++         <li>Fixed invalid opertions on Wayland for OpenGL ES3 capable devices</li>
++         <li>Use OpenGL ES3 by default on Linux</li>
++         <li>New OSM data as of July 18</li>
++         <li>Fixed/updated API (see omaps.app/api)</li>
++         <li>Added missing shop types from top OSM tags</li>
++         <li>Minor routing improvement</li>
++         <li>Use min_level for 3D buildings</li>
++         <li>Allow removing cuisines from selection in the Editor</li>
++         <li>Fixed be, de, ru, uk translations</li>
++         <li>Fixed FB URLs validation</li>
```

Comment:
These 3 lines are for mobile devices only

Reply:
Although this flatpak does run on certain mobile devices, I have removed these three lines.
If we are to use your full relevant list, then I think it is better to start a new PR in the upstream repo.

Reply:
Our [android] and [ios] changes are not relevant at all for the Linux version. Let's not confuse users )

Reply:
OK